index.html

```diff
@@ -1,72 +1,82 @@
-  <head>
+<!--[if lte IE 6]>
+<html class="preIE7 preIE8 preIE9">
+   <![endif]-->
+<!--[if IE 7]>
+   <html class="preIE8 preIE9">
+      <![endif]-->
+<!--[if IE 8]>
+      <html class="preIE9">
+         <![endif]-->
+<!--[if gte IE 9]><!-->
+<html>
+<!--<![endif]-->
+
+<head>
     <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-    <title>My ReDi Progject</title>
-  <link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
-  <link rel="stylesheet" href="stylesheet.css" type="text/css">
-  </head>
-  <body>
-      
-      <header>
-          <nav> 
-                <a href="">HOME</a>
-                <a href="">ABOUT ME</a>  
-                <a href="index.html" class="current">BASE PAGE</a>
-          </nav>
-      </header>
-      <div class="main">        
-          <section class="post">
-              <h2>Traveling in Bavaria</h2>
-                 <div>
-              <img class="postpic" src="travel.jpg" >
-              </div>
-              <div class="description">
-                  What I miss: I love traveling and miss the nature and landscapes that I see while traveling.
-              </div>
-              <div class="link">
-              <a href="https://goo.gl/maps/Vx9rXPP3NdG63ysq7" target="_blank">location</a>
-              </div>
-          </section>
-          
-             <section class="post">
-              <h2>Tierpark Hellabrunn</h2>
-                 <div>
-              <img class="postpic" src="tierpark.jpg" width=100%>
-              </div>
-              <div class="description">
-                  What I miss: I miss the moments we spent there with my family. Our tickets will end on April 16 :(
-              </div>
-              <div class="link">
-              <a href="https://goo.gl/maps/XMUj6szEoj6F7ufn6" target="_blank">location</a>
-              </div>
-          </section>
-          
-             <section class="post">
-              <h2>Deutsches Museum Verkehrszentrum</h2>
-                 <div>
-              <img class="postpic" src="museum.jpg" width=100%>
-              </div>
-              <div class="description">
-                  What I miss: My son loves this museum, and I miss the time I could spend here with him.
-              </div>
-              <div class="link">
-              <a href="https://goo.gl/maps/WvBUTQBVMSymd4UCA" target="_blank">location</a>
-              </div>
-          </section>
-          
-         
-        
-          
-          
-      </div>
-       
-  <footer>
-      made by Sergei Golovnev for REDI: Frontend Development 2
-      </footer>
-    </body>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>My ReDi Project</title>
+    <link rel="stylesheet" href="stylesheet.css" type="text/css">
+</head>
+<body>
+    <header>
+        <ul>
+            <li><a id="menu_burger"><img src="burger_button_w.png" height="15px"></a></li>
```

Comment:
here you can use fontAwsome icons. for that you would need to include this
`<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">`
and then usage look like that:
`<button><i class="fa fa-bars"></i></button>`
More info here https://fontawesome.com/

Reply:
Thank you! That's very convinient! Fixed!